source/frontend/src/hooks/pims-api/useApiAcquisitionFile.ts

```diff
@@ -42,6 +42,19 @@ export const useApiAcquisitionFile = () => {
             Accept: 'application/vnd.ms-excel',
           },
         }),
+      exportAcquisitionFiles: (
+        filter: IPaginateAcquisition,
+        outputFormat: 'csv' | 'excel' = 'excel',
```

Comment:
same comment here regarding csv, pretty sure this will throw an error from the backend if you request csv.